entry_types/scrolled/package/src/frontend/assets/images/playerControls/check_48px.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="48" height="48" viewBox="0 0 48 48"><path d="M18 32.34L9.66 24l-2.83 2.83L18 38l24-24-2.83-2.83z"/></svg>
```

Comment:
In case the 48px/24px variants differ only by size, `react-svg-loader` [supports specifying width/height](https://github.com/boopathi/react-svg-loader/tree/master/packages/react-svg-loader#usage) on the imported component.

Reply:
This was just a remnant from initially adding the icons. While I was at it, I downloaded both sizes. Doesn't look like we need the 48px variants, though, so i removed them now.